tests/unittests/Makefile

```diff
@@ -48,8 +48,8 @@ BOARD_INSUFFICIENT_MEMORY := airfy-beacon \
                              nz32-sc151 \
                              opencm904 \
                              openmote \
-                             openmote-cc2538 \
                              openmote-b \
+                             openmote-cc2538 \
```

Comment:
These changes are unrelated. If you want to sort them, please put that in a separate PR.